### PR TITLE
Simple list of resources

### PR DESCRIPTION
**Pull request by kilchenmann**
https://dasch.myjetbrains.com/youtrack/issue/DSP-459

### CODE REVIEW

**Comment by kilchenmann** [comment]
Waiting for [PR 215 in js-lib](https://github.com/dasch-swiss/dsp-js-lib/pull/215)

**Comment by kilchenmann** [comment]
Waiting for PR #127

File: projects/dsp-ui/src/lib/viewer/views/list-view/resource-list/resource-list.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { MatLineModule } from '@angular/material/core';
+import { MatIconModule } from '@angular/material/icon';
+import { MatListModule } from '@angular/material/list';
+import { MockResource, ReadResourceSequence } from '@dasch-swiss/dsp-js';
+import { DspActionModule } from 'projects/dsp-ui/src/lib/action';
+import { ResourceListComponent } from './resource-list.component';
+
+/**
+ * Test host component to simulate child component, here property-view.
```
Comment (flavens) [inline projects/dsp-ui/src/lib/viewer/views/list-view/resource-list/resource-list.component.spec.ts:11]: "here property-view" -> to update!!

File: projects/dsp-ui/src/lib/viewer/views/list-view/resource-list/resource-list.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+    beforeEach(() => {
+        testHostFixture = TestBed.createComponent(TestParentComponent);
+        testHostComponent = testHostFixture.componentInstance;
+        testHostFixture.detectChanges();
+
+        expect(testHostComponent).toBeTruthy();
+    });
+
+    it('expect 2 resources', () => {
```
Comment (flavens) [inline projects/dsp-ui/src/lib/viewer/views/list-view/resource-list/resource-list.component.spec.ts:80]: the title should be updated: 2 > 5

Comment (kilchenmann) [reply projects/dsp-ui/src/lib/viewer/views/list-view/resource-list/resource-list.component.spec.ts]: solved in 7fd9d73

File: projects/dsp-ui/src/lib/viewer/views/list-view/resource-list/resource-list.component.spec.ts (hunk 1 above)
Comment (kilchenmann) [reply projects/dsp-ui/src/lib/viewer/views/list-view/resource-list/resource-list.component.spec.ts]: solved in 7fd9d73

**Comment by kilchenmann** [comment]
@flavens What do you think? Can we do a review on this PR soon again? Thanks

File: projects/dsp-ui/src/lib/viewer/views/list-view/resource-list/resource-list.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    });
+
+    it('should open first resource', () => {
+        // trigger the click
+        const nativeElement = testHostFixture.nativeElement;
+        const item = nativeElement.querySelector('mat-list-item');
+        item.dispatchEvent(new Event('click'));
+
+        spyOn(testHostComponent, 'openResource').call('http://rdfh.ch/0001/H6gBWUuJSuuO-CilHV8kQw');
+        expect(testHostComponent.openResource).toHaveBeenCalled();
```
Comment (mdelez) [inline projects/dsp-ui/src/lib/viewer/views/list-view/resource-list/resource-list.component.spec.ts:92]: I think this is redundant as you are also checking that it is called once below

Comment (kilchenmann) [reply projects/dsp-ui/src/lib/viewer/views/list-view/resource-list/resource-list.component.spec.ts:92]: Yes, I thought so too.

File: projects/dsp-ui/src/lib/viewer/views/list-view/resource-list/resource-list.component.scss [hunk 4]
```diff
@@ -0,0 +1,43 @@
+// TODO: link class should be moved to root level of the module
```
[the remark this answers is not in the document]
Comment (kilchenmann) [reply projects/dsp-ui/src/lib/viewer/views/list-view/resource-list/resource-list.component.scss:1]: Thanks. We can remove the comment, when we update your PR after merging this one.

**Pull request merged**